noxfile.py

```diff
@@ -8,6 +8,7 @@
 
 import nox
 
+nox.options.error_on_missing_interpreters = True
```

Comment:
Can you please remove `--error-on-missing-interpreters` in `.github/workflows/ci.yml` then?